static/js/components/RepeatableContentListing.test.tsx

```diff
@@ -254,6 +255,31 @@ describe("RepeatableContentListing", () => {
     ).toBe(false)
   })
 
+  test("should filter based on query param", async () => {
+    helper.browserHistory.push("/?q=search")
+    const { wrapper } = await render()
+    contentListingItems.forEach((item, idx) => {
+      const li = wrapper.find("li").at(idx)
+      expect(li.text()).toContain(item.title)
+    })
+  })
+
+  test("should let the user filter via text input", async () => {
+    const { wrapper } = await render()
+    const filterInput = wrapper.find(".site-search-input")
+    const event = {
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      preventDefault() {},
```

Comment:
**Random question:** I noticed `useTextInputState` in your hook, so I went to go look at it... do you know why we call `preventDefault` in `useTextInputState`? When is preventing default on a text input important?